tests/integration/tables/system_extensions.cpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+// Sanity check integration test for system_info
+// Spec file: specs/system_info.table
+
+#include <osquery/tests/integration/tables/helper.h>
+
+namespace osquery {
+namespace table_tests {
+
+class SystemExtension : public testing::Test {
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
+};
+
+TEST_F(SystemExtension, test_sanity) {
+  QueryData data = execute_query("select * from system_extensions");
```

Comment:
Test might fail on macOS 10.14, I have a VM, let me know if you want me to test there

Reply:
> Test might fail on macOS 10.14, I have a VM, let me know if you want me to test there

Disregard this. It built and tested fine on macOS 10.14.